contrib/jmx-metrics/src/main/java/io/opentelemetry/contrib/jmxmetrics/GroovyMetricEnvironment.java

```diff
@@ -62,22 +79,34 @@ public GroovyMetricEnvironment(
       final JmxConfig config,
       final String instrumentationName,
       final String instrumentationVersion) {
-    meter = OpenTelemetry.getGlobalMeter(instrumentationName, instrumentationVersion);
+    longUpdaterRegistry = new ConcurrentHashMap<>();
```

Comment:
Another option would be to initialize these where they are declared. Could get rid of a couple of lines of code :-)

Reply:
Thanks, no need to wait til the constructor.